bloom/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM coreweave/nccl-tests:2022-11-06_19-21-22.11_EST
+
+# setup python and conda
+
+RUN apt-get install -y --no-install-recommends \
+    python3 python3-dev python3-pip git libssl-dev pkg-config
+
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
+    bash Miniconda3-latest-Linux-x86_64.sh -b -p /opt/conda && \
+    rm Miniconda3-latest-Linux-x86_64.sh
+
+ENV PATH=/opt/conda/bin:$PATH
+
+RUN conda create -y -n tr11-176B-ml python=3.8
+
+SHELL ["conda", "run", "-n", "tr11-176B-ml", "/bin/bash", "-c"]
+
+RUN pip install transformers && \
+    pip install --pre torch torchvision torchaudio -f https://download.pytorch.org/whl/test/cu115/torch_test.html -U
```

Comment:
@harubaru You might want to match the CUDA versions (11.7 or 11.6) with Torch CUDA

Reply:
`conda install pytorch torchvision torchaudio pytorch-cuda=11.7 -c pytorch -c nvidia`

Reply:
Is there any possibility to use `pip` instead? Using conda to install pytorch breaks the step for installing apex

Reply:
Yup that works.